src/estop.cpp

```diff
@@ -37,6 +41,26 @@ void sigintShutdownNode(int sig)
 {
     gpioSetValue(ESTOP_PIN, 0);
     ros::shutdown();
+    g_request_shutdown = 1; // Set flag
+}
+
+/**
+ * This function is called before the node is killed using rosnode kill. It disables the estop and shutdown the node. 
+*/
+void shutdownCallback(XmlRpc::XmlRpcValue& params, XmlRpc::XmlRpcValue& result)
+{
+  int num_params = 0;
+  if (params.getType() == XmlRpc::XmlRpcValue::TypeArray)
```

Comment:
I'm not a fan of implicit braces. But the ROS C++ Standard allow it, so I'll leave it up to you.

Reply:
Addressed to match the rest of our codebase in commit [093f6dd](https://github.com/clubcapra/capra_estop/pull/3/commits/093f6dd730abc3697618692cc945e137590240b9)